RUFAS/routines/field/crop/non_water_uptake.py

```diff
@@ -0,0 +1,171 @@
+from abc import ABC
+from typing import Optional
+
+from RUFAS.routines.field.crop.crop_data import CropData
+from RUFAS.routines.field.crop.nutrient_uptake import NutrientUptake
+from RUFAS.routines.field.soil.soil_data import SoilData
+
+
+class NonWaterUptake(NutrientUptake, ABC):
```

Comment:
```suggestion
class NonWaterUptake(NutrientUptake):
```
Non water uptake does not need to inherit the Abstract Base Class, because it doesn't have any unique abstract methods.